beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/historical/HistoricalBatchFetcher.java

```diff
@@ -224,7 +224,7 @@ private SafeFuture<Boolean> requestBlocksAndBlobSidecarsByRange() {
             requestParams.getStartSlot(), requestParams.getCount(), requestManager::processBlock);
 
     final SafeFuture<Void> blobSidecarsRequest;
-    if (blobSidecarManager.isAvailabilityRequiredAtSlot(endSlot)) {
+    if (blobSidecarManager.isStorageOfBlobSidecarsRequiredAtSlot(endSlot)) {
```

Comment:
The problem I see here is that we could stop backfilling blocks too if we struggle finding peers serving old blobs. But this is a limitation of current sync design, which bonds block and blobs sync.

Secondly we should make sure we don't downscore peers not serving blobs outside spec DA.